screen-reader/ios/Plugin/ScreenReaderPlugin.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import Capacitor
+
+/**
+ * Please read the Capacitor iOS Plugin Development Guide
+ * here: https://capacitorjs.com/docs/plugins/ios
+ */
```

Comment:
```suggestion

```
This should be removed, it's there to help people get started when they develop plugins, but should not be part of the final plugin